src/services/firebase.js

```diff
@@ -66,4 +65,9 @@ export const startFirebaseUI = function (elementId) {
         ui.start(elementId, uiConfig)
     })
 };
-export const db = firebase.database();
\ No newline at end of file
+
+export function getCurrentUser() {
+    return firebase.auth().currentUser;
+}
+
+export default firebase.database();
```

Comment:
we should not export this as default, since it's not intuitive that we import this object specifically if we were to do something like ```import x from 'firebase.js'```.